naomi/orderly-web.yml

```diff
@@ -89,7 +89,7 @@ web:
   image:
     repo: vimc
     name: orderly-web
-    tag: master
+    tag: mrc-2951
```

Comment:
Into whereever you're keeping track (if anywhere) of smoothing this type of setup over, it might be worth adding something about reducing this duplication